src/test/regress/expected/with_basics.out

```diff
@@ -220,6 +220,88 @@ JOIN
   users_table USING (user_id);
 ERROR:  cannot push down this subquery
 DETAIL:  Aggregates without group by are currently unsupported
+-- column aliases in CTE
+WITH top_ten(id, val1) AS (
+	SELECT user_id, value_1 FROM users_table ORDER BY value_1 DESC, user_id DESC LIMIT 10
+)
+SELECT * FROM top_ten;
```

Comment:
I wonder if we need `ORDER BY` on the top as well? What do you think? I think given that there is a single CTE, it seems fine. Feel free to ignore this comment.

Reply:
we have order by in another test
```sql
-- computed targets from columns and group by outside
WITH top_ten(id, val, val_mul, val_sum) AS (
	SELECT user_id, value_1, value_1*2, value_1 + value_2 FROM users_table ORDER BY value_1 DESC, user_id DESC
)
SELECT id, count(*), avg(val), max(val_mul), min(val_sum) FROM top_ten
GROUP BY id
ORDER BY 2 DESC, 1 DESC;
```

Reply:
I mean could the results be inconsistent since we skip `ORDER BY`

Reply:
it should not cause inconsistency, we have order by in inner query on all columns. Generated intermediate table should have the results in order. If that behavior changes for any reason we would know from this test